lib/rules/at-rule-descriptor-value-no-invalid/__tests__/index.mjs

```diff
@@ -0,0 +1,89 @@
+import rule from '../index.mjs';
+const { messages, ruleName } = rule;
+
+testRule({
+	ruleName,
+	config: true,
+
+	accept: [
+		{
+			code: 'a { color: 0; }',
+			description: 'ignore normal values',
+		},
+		{
+			code: '@counter-style foo { system: cyclic; symbols: url(bar.svg); suffix: " "; }',
+		},
+		{
+			code: '@property --foo { syntax: "<integer>"; inherits: false; initial-value: 0; }',
+		},
+		{
+			code: '@view-transition { navigation: auto; }',
+		},
+		{
+			code: '@foo { bar: 0 }',
+		},
+		{
+			code: '@media (width > 400px) { a { color: red; } }',
+		},
+		{
+			code: 'a { @media (width > 400px) { color: red; } }',
+		},
+	],
+
+	reject: [
+		{
+			code: '@counter-style foo { system: bar; }',
+			message: messages.rejected('system', 'bar'),
+			line: 1,
+			column: 30,
+			endLine: 1,
+			endColumn: 33,
+		},
+		{
+			code: '@property --foo { syntax: bar; }',
+			message: messages.rejected('syntax', 'bar'),
+			line: 1,
+			column: 27,
+			endLine: 1,
+			endColumn: 30,
+		},
+		{
+			code: '@property --foo { syntax: bar; inherits: baz; }',
+			warnings: [
+				{
+					message: messages.rejected('syntax', 'bar'),
+					line: 1,
+					column: 27,
+					endLine: 1,
+					endColumn: 30,
+				},
+				{
+					message: messages.rejected('inherits', 'baz'),
+					line: 1,
+					column: 42,
+					endLine: 1,
+					endColumn: 45,
+				},
+			],
+			description: 'multiple invalid value in @property',
+		},
+		{
+			code: '@property --foo { syntax: cyclic; }',
+			message: messages.rejected('syntax', 'cyclic'),
```

Comment:
```suggestion
			code: '@property --foo { system: cyclic; }',
			message: messages.rejected('system', 'cyclic'),
```

ref https://developer.mozilla.org/fr/docs/Web/CSS/@counter-style/system#cyclic

Reply:
As the description says, "value known for another at-rule but not @\property," I am purposely using a different value.

Reply:
I know and that value is `system` but for the test to be working—i.e. the pair property/value to be valid—the property must exist on another at rule.

Reply:
`system: cyclic` is used in the [accept test](https://github.com/stylelint/stylelint/pull/8211/files#diff-d9fc880b1da150e8a06381ed5af062cf0934a61e0004fb7fe98a26cd88bd1660R14). 
Is this not good enough?

Reply:
If you don't want to change the `code` and `message` properties you will have to change the `description`.
I can't give you one because I don't know what you are trying to test if it's not a regression test that verify that other pairs aren't passing for non-matching at rules—which is the all point of the rule.

Reply:
I think the test is fine. It's similar to the `@property --foo { syntax: bar; }` test but uses a value known to another descriptor rather than something made up like `bar`.

However, this discussion brings up a good point. Let's add an _accept_ `test` for:

```css
@property --foo { system: bar; }
```

With the description: "a descriptor (with an unknown value) that is unknown to the at-rule"

As checking for unknown descriptors is the responsibility of the `at-rule-descriptor-no-unknown` rule: https://github.com/stylelint/stylelint/pull/8197.